website/views/options/social/group.jade

```diff
@@ -82,7 +82,7 @@ a.pull-right.gem-wallet(ng-if='group.type!="party"', popover-trigger='mouseenter
                     span.glyphicon.glyphicon-ban-circle(tooltip=env.t('banTip'))
                 a.media-body
                   span(ng-click='clickMember(member._id, true)')
-                    | {{member.profile.name}}
+                    | {{member.profile.name}} (<strong>{{member.stats.hp.toFixed(1)}}</strong> #{env.t('hp')})
```

Comment:
It would need to be:

```jade
| {{member.profile.name}} (
| strong {{member.stats.hp.toFixed(1)}}
| #{env.t('hp')})
```

or

```jade
| {{member.profile.name}} (#[strong {{member.stats.hp.toFixed(1)}}] #{env.t('hp')})
```